doc/man7/provider.pod

```diff
@@ -268,6 +268,14 @@ algorithm identifier to the appropriate fetching function.
 The default provider is built in as part of the F<libcrypto> library.
 Should it be needed (if other providers are loaded and offer
 implementations of the same algorithms), the property "provider=default"
+can be used as a search criterion for these implementations.  The default
+provider includes all the functionality of the serialization provider below.
+
+=head2 Serialization provider
+
+The serialization provider is built in as part of the F<libcrypto> library.
+Should it be needed (if other providers are loaded and offer
+implementations of the same algorithms), the property "provider=serialization"
```

Comment:
Drive by comment: base?

Reply:
Yeah, should have been "base" in several places in this paragraph.  All fixed now.